### PR TITLE
CORDA-2888 Allow bring-your-own-config to docker image

### PR DESCRIPTION
**Pull request by StefanIliev545**
modify run logic so that if node config is present, do not generate a new one

### CODE REVIEW

File: docker/src/bash/generate-config.sh [hunk 1]
```diff
@@ -47,16 +44,15 @@ function generateGenericCZConfig(){
...
         java -Djava.security.egd=file:/dev/./urandom -Dcapsule.jvm.args="${JVM_ARGS}" -jar /opt/corda/bin/corda.jar \
-                --initial-registration \
-                --base-directory /opt/corda \
-                --config-file ${CONFIG_FOLDER}/node.conf \
-                --network-root-truststore-password ${NETWORK_TRUST_PASSWORD} \
-                --network-root-truststore ${CERTIFICATES_FOLDER}/${TRUST_STORE_NAME} &&\
-        echo "Succesfully registered with ${DOORMAN_URL}, starting corda" && \
+                initial-registration \
+                --base-directory=/opt/corda \
+                --config-file=/etc/corda/node.conf \
```
Comment (roastario) [inline docker/src/bash/generate-config.sh]: this should be ${CONFIG_FOLDER}

Comment (roastario) [inline docker/src/bash/generate-config.sh]: this should be ${CONFIG_FOLDER}

File: docker/src/bash/generate-config.sh [hunk 2]
```diff
@@ -47,16 +44,15 @@ function generateGenericCZConfig(){
...
-
+    fi
         java -Djava.security.egd=file:/dev/./urandom -Dcapsule.jvm.args="${JVM_ARGS}" -jar /opt/corda/bin/corda.jar \
-                --initial-registration \
-                --base-directory /opt/corda \
-                --config-file ${CONFIG_FOLDER}/node.conf \
-                --network-root-truststore-password ${NETWORK_TRUST_PASSWORD} \
-                --network-root-truststore ${CERTIFICATES_FOLDER}/${TRUST_STORE_NAME} &&\
-        echo "Succesfully registered with ${DOORMAN_URL}, starting corda" && \
+                initial-registration \
```
Comment (anthonykeenan) [inline docker/src/bash/generate-config.sh]: Isn't changing this back to initial-registration undoing an earlier change?

Comment (StefanIliev545) [reply docker/src/bash/generate-config.sh]: It is the same as what was merged in ENT. My search on looking for that was
 names = ["--initial-registration"],
            description = ["DEPRECATED. Starts initial node registration with Corda network to obtain certificate from the permissioning server."],

File: docker/src/bash/generate-config.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (StefanIliev545) [reply docker/src/bash/generate-config.sh]: Thats the outdated version of the file

File: docker/src/bash/generate-config.sh (hunk 2 above)
Comment (anthonykeenan) [reply docker/src/bash/generate-config.sh]: Yes but we merged a PR into OS that allowed for the docker image to be used with older versions of Corda - that wasn't taken into ENT. But I don't want to revert that change.

**Pull request merged**